applications/asset_tracker/src/cloud_codec/cloud_codec.c

```diff
@@ -366,7 +366,7 @@ int cloud_encode_env_sensors_data(const env_sensor_data_t *sensor_data,
 
 	char buf[6];
 	uint8_t len;
-	struct cloud_channel_data cloud_sensor;
+	struct cloud_channel_data cloud_sensor = {0};
```

Comment:
Could have spacing after `{` and before `}` to align with rest of file, but doesn't seem to bother checkpatch to leave it like this